src/datasources/asset/AssetCoordonatorDataSource.test.ts

```diff
@@ -7,14 +7,14 @@ import {
   requestMatching,
   setupDataSource,
 } from "test/fixtures";
-import { AssetDataSource } from "./AssetDataSource";
-import { AssetMetadataQuery } from "./types";
+import { AssetCoordonatorDataSource } from "./AssetCoordonatorDataSource";
+import { ListAssetsQuery, AssetQueryType } from "./types";
 import { AssetPresenceWithSystemConnectionModel, AssetsResponse } from "datasources/asset-common/types";
 
-let ds: AssetDataSource, backendSrv: MockProxy<BackendSrv>
+let ds: AssetCoordonatorDataSource, backendSrv: MockProxy<BackendSrv>
```

Comment:
I don't think it's necessary to add `Coordonator` everywhere in the name. I'd keep it simply AssetDataSource as it was in the beginning